app/views/hearings/show.html.haml

```diff
@@ -18,10 +24,19 @@
       - if @hearing_events
```

Comment:
existing hearing_events code breaks if 
HEARING_EVENTS is **false**
HEARING_DATA is **true**

Seems like hearing_events and hearing_data are coupled. When hearing_data is true then hearing_events is still assumed to be there when it shouldn't.